release-notes/README.md

```diff
@@ -8,6 +8,20 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 ## Project Griffon will be Assurance!
 We're pleased to announce that Project Griffon will be generally available to all Adobe Experience Cloud customers as Assurance. To learn more about this transition see [here](../../beta/project-griffon/).
 {% endhint %}
+## October 26, 2022
+
+### Flutter AEPEdge 1.0.0
+
+* [Edge Network extension](https://pub.dev/packages/flutter_aepedge) is now supported in Flutter.
+
+### Flutter AEPEdgeIdentity 1.0.0
+
+* [Identity for Edge Network extension](https://pub.dev/packages/flutter_aepedgeidentity) is now supported in Flutter. 
+
+### Flutter AEPEdgeConsent 1.0.0
+
+* [Consent for Edge Network extension](https://pub.dev/packages/flutter_aepedgeconsent) is now supported in Flutter.
```

Comment:
let's group them together under a single header 
### AEP Flutter plugins for Edge Network
The following AEP Flutter plugins have been published:
- [flutter_aepedge@1.0.0](https://pub.dev/packages/flutter_aepedge)
- ...
- 
-e.g. https://aep-sdks.gitbook.io/docs/release-notes#aep-react-native-plugins